__tests__/api.test.js

```diff
@@ -251,5 +251,56 @@ describe('app', () => {
             })
           })
         })
+    })
+
+    describe("7. GET /api/reviews/:review_id/comments", () => {
+    test('should return an array of comment objects relating to review_id', () => {
+      return request(app)
+        .get('/api/reviews/3/comments')
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments).toHaveLength(3);
+          comments.forEach((comment) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                body: expect.any(String),
+                review_id: expect.any(Number),
+                author: expect.any(String),
+                votes: expect.any(Number),
+                created_at: expect.any(String)
+              })
+            )
+          })
+        })
       })
+      test('404; handles incorrect id', () => {
+        return request(app)
+        .get('/api/reviews/50/comments')
+        .expect(404)
+        .then(({body: { msg }}) => {
+          expect(msg).toBe('No comment found for review_id: 50')
+        })
+      })
+      test('404; handles id with no comments', () => {
+        return request(app)
+        .get('/api/reviews/1/comments')
+        .expect(404)
+        .then(({body: { msg }}) => {
+          expect(msg).toBe('No comment found for review_id: 1')

```

Comment:
would we expect an 200 for an article that exists but has no comments? Think about how this information would come back, maybe as an empty array?